lib/assessment.rb

```diff
@@ -13,28 +15,30 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(0) do |memo,item|
```

Comment:
Yep, that's correct. You could also have written it as 
`enum.reduce(0, &:+)` (or just `enum.reduce(0, :+)`, as of Ruby 2.0)